pages/platform/data-processing/38_HOWTO_use-with-ovh-api/guide.en-asia.md

```diff
@@ -0,0 +1,99 @@
+
+To store and use them you can create file or set them as environment variables. 
+The way you proceed depends on which medium you chose to reach the API.
+
+But most of the wrappers provided by OVHcloud use an `ovh.conf` file that looks like the following:
+    ```
+    endpoint=ovh-eu
+    application_key=my_app_key
+    application_secret=my_application_secret
+    consumer_key=my_consumer_key
+    ```
+You should create in your application's directory and fill it with your own 3 keys.
+
+### Step 2: Chose your mean to reach the OVHcloud API
+Now you got your credentials, your able to query the OVHcloud API. To do so, you can chose between: 
+- using a http client or the curl command line. 
+You can find a documentation to learn how to use the OVHcloud API [here](../../../account/customer/first-steps-with-ovh-api).
+
+- using one of the wrappers provided by OVHcloud which will handle all the authentication for you.
+The several wrappers (for Python, java, c#, PHP, golang, etc...) are available on the [OVHcloud github](https://github.com/ovh?q=&type=&language=&sort=){.external}.
+All the information needed to use them are in their respective README.md files.
+
+### Step 3: Have a look to the OVHcloud API endpoints
+In order to know which endpoints are available through the OVHcloud API, you can have a look at this [web page](https://api.ovh.com/){.external} and click on the explore the API button.
+
+To use this UI, you have to be logged in with your OVHcloud account.
+
+There you can find all the Data Processing endpoints in the "cloud" section. 
+![Raw tab in UI](images/cloud.png){.thumbnail}
+Scroll down into the "cloud section until you reach the `/cloud/project/{serviceName}/dataProcessing/... ` endpoints.
+Once you get there, you can try out the endpoints directly in the UI by clicking on them.
+
+>[!primary]
+>
+> The serviceName parameters that is required for every endpoint of the `cloud` section is your Public Cloud project ID.
+
+If you need further information about an endpoint, you can find the response class that tells you what the API response will look like, just under the `execute` button, and examples of wrappers code.
+![Response class tab in UI](images/response.png){.thumbnail}
+
+You can also see what your request body looks like in a json format for a specific endpoint by having a look in the `raw` tab that appears if the request has been executed once.
+![Raw tab in UI](images/raw.png){.thumbnail}
+
+>[!primary]
+>
+> For the POST on `cloud/project/{serviceName}/dataProcessing/jobs` endpoint, the `engineParameters` property is a mandatory list of key/value parameters.
+> To know which parameters you should use, you just have to query the `cloud/project/{serviceName}/dataProcessing/capabilites` endpoint to list the several parameters depending on the engine you are using.
+> It will also show you which of those parameters are required and how they should be formed.
+
+
+### Step 4: Make sure your project is activated
```

Comment:
Make sure the Data Processing service is activated for your Public Cloud project